Source/WebKit/UIProcess/API/Cocoa/WKUIDelegatePrivate.h

```diff
@@ -208,6 +208,9 @@ struct UIEdgeInsets;
 
 - (void)_webView:(WKWebView *)webView decidePolicyForModalContainer:(_WKModalContainerInfo *)containerInfo decisionHandler:(void (^)(_WKModalContainerDecision))decisionHandler WK_API_AVAILABLE(macos(13.0), ios(16.0));
 
+- (void)_webView:(WKWebView *)webView updatedAppBadge:(NSNumber *)badge WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA));
```

Comment:
I don't think the webView updated the app badge. aren't we asking the app to update the app badge?